tests/integration/test_zos_jobs.py

```diff
@@ -32,6 +32,14 @@ def test_list_jobs_should_return_valid_spool_information(self):
         """Executing the list_jobs method should return a list of found jobs in JES spool."""
         command_output = self.jobs.list_jobs(owner=self.jobs_fixtures_json['TEST_JCL_OWNER'])
         self.assertIsInstance(command_output, list)
+    
+    def test_change_job_class(self):
+        """Execute the change_jobs_class should execute successfully."""
+        execution_output = self.jobs.submit_from_mainframe(self.jobs_fixtures_json['TEST_JCL_MEMBER'])
+        jobname = execution_output['jobname']
+        jobid = execution_output['jobid']
+        command_output = self.jobs.change_jobs_class(jobname, jobid, "A")
+        self.assertIsNotNone(command_output['jobid'])
```

Comment:
This is good to verify that sensible command output was returned 👍 

Could you add a check that the job class was changed? To do this we'll need to call `self.jobs.get_job_status` after changing the job class, and verify that `command_output["class"]` matches the new class.